tests/app/main/test_frameworks.py

```diff
@@ -3646,11 +3645,9 @@ def _send_email(self, clarification_question):
             data={'clarification_question': clarification_question}
         )
 
-    def _assert_clarification_email(self, send_email, is_called=True, succeeds=True):
-
-        if succeeds:
-            assert send_email.call_count == 2
```

Comment:
Why did it used to get called more than once??

Reply:
It sends two emails - one to admins notifying of a new clarification question, and then a confirmation email to the supplier.

We have only converted one of the two at this point, so there's still one email getting sent with mandrill.

Reply:
Can we maybe rename `send_email` to `mandrill_send_email`, if I'm understanding correctly what that means then. But yep.

Reply:
Ah yep sure, will do that.